README.md

```diff
@@ -1,36 +1,138 @@
+</p>
+<br>
 
-## Installation
+**napari** is a fast interactive multi-dimensional image viewer for Python. It's designed for browsing, annotating, and analyzing large multi-dimensional images. It's built on top of `PyQt` (for the GUI), `vispy` (for performant GPU-based rendering), and the scientific Python stack (`numpy`, `scipy`).
 
-napari can be installed on most operating systems with `pip install napari`.
+We're developing **napari** in the open! But the project is in a **pre-alpha** stage. You can follow progress on this repository, test out new versions as we release them, and contribute ideas and code. Expect **breaking changes** from patch to patch.
 
-### Downloading OpenGL
+## installation
 
-Windows and Linux users may need to install the [Mesa OpenGL](https://www.mesa3d.org/) library.
+napari can be installed on most Mac OS X and Linux systems with Python 3.6 or 3.7 by calling 
 
-`vispy` provides an easily-downloadable (but out-of-date) Windows version for [64-](https://github.com/vispy/demo-data/raw/master/mesa/opengl32_mingw_64.dll) and [32-bit](https://github.com/vispy/demo-data/raw/master/mesa/opengl32_mingw_64.dll) operating systems.
+```sh
+$ pip install napari
+```
 
-Linux users can complete the installation process from the command line:
+We're working on adding Windows support.
 
-```sh
-$ sudo apt update
-$ sudo apt install libgl1-mesa-glx
+## simple example
+
+From inside the IPython shell or a Jupyter notebook you can run the following to open up an interactive viewer
+
+```python
+%gui qt5
+from skimage import data
+from napari import ViewerApp
+viewer = ViewerApp(data.astronaut())
+```
+
+![image](resources/screenshot-add-image.png)
+
+To do the same thing inside a script call
+
+```python
+from skimage import data
+from napari import ViewerApp
+from napari.util import app_context
+
+with app_context():
+	viewer = ViewerApp(data.astronaut())
+```
+
+## more features
+
+Check out the scripts in the `examples` folder to see some of the functionality we're developing!
+
+For example, you can add multiple images in different layers and adjust them
+
+```python
+from skimage import data
+from skimage.color import rgb2gray
+from napari import ViewerApp
+from napari.util import app_context
+
+with app_context():
+    viewer = ViewerApp(astronaut=rgb2gray(data.astronaut()),
+                       photographer=data.camera(),
+                       coins=data.coins(),
+                       moon=data.moon())
+
+    viewer.layers.remove('coins')
+    viewer.layers['astronaut', 'moon'] = viewer.layers['moon', 'astronaut']
+```
+
+![image](resources/screenshot-layers.png)
+
+You can add markers on top of an image
+
+```python
+from skimage import data
+from skimage.color import rgb2gray
+from napari import ViewerApp
+from napari.util import app_context
+
+with app_context():
+    viewer = ViewerApp()
+    viewer.add_image(rgb2gray(data.astronaut()))
+    points = np.array([[100, 100], [200, 200], [333, 111]])
```

Comment:
Do you think it's helpful to add comments to describe how the data are structured (e.g., the order of the axes for the images and/or the shape of the points and size arrays).

Reply:
cool idea, added a few comments for the less obvious examples